lazylink.py

```
@@ -15,13 +15,15 @@
 # TODO: Use API to search for bugs assigned in the past week?
 def main():
     args = set_and_parse_args()
+
     for f in args.files:
         lazylink_file(f)
 
 def set_and_parse_args():
     # TODO: ArgumentParser desc.
     parser = ArgumentParser()
-    parser.add_argument('files', type=file, nargs='*')
+    parser.add_argument('files', type=file, nargs='+')
```

Since you need one argument to do anything you should throw an error if there isn't at least one instead of doing nothing.